src/dune_rules/melange_rules.ml

```diff
@@ -16,43 +16,43 @@ let ocaml_flags sctx ~dir melange =
     with_vendored_flags ~ocaml_version flags
   | false -> flags
 
-let lib_output_dir ~target_dir ~lib_dir =
+let lib_output_dir ~target_dir lib =
+  let info = Lib.info lib in
+  let lib_dir =
+    match Lib.Local.of_lib lib with
+    | None ->
+      let package_name = Option.value_exn (Lib_info.package info) in
+      let bctx, _ = Path.Build.extract_build_context_dir_exn target_dir in
+      Path.Build.(
+        relative
+          ((relative bctx) "node_modules")
```

Comment:
We have `Path.Build.L.relative` to make this easier to read

Reply:
Added in https://github.com/anmonteiro/dune/pull/3/commits/4224f26abfa3f8094ab25d7cccb852f7d9065a24, can be merged separately or together with https://github.com/anmonteiro/dune/pull/3.